python/paddle/quantization/base_observer.py

```diff
@@ -0,0 +1,34 @@
+"""Abstract observer class."""
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+
+from .base_quanter import BaseQuanter
+
+__all__ = ["BaseObserver"]
```

Comment:
`BaseObserver` is already in \_\_all\_\_ of `python/paddle/quantization/__init__.py`, and can not be here.

Reply:
Fixed. Thx.